scripts/west_commands/create_board/ncs_create_board.py

```diff
@@ -16,6 +16,12 @@
 except ImportError:
     from yaml import Loader
 
+sys.path.append(Path(__file__).parents[2])
```

Comment:
It seems like `utils` is not found correctly. I managed to run ncs-create-board with this change.
```suggestion
sys.path.append(str(Path(__file__).parents[1]))
```

Reply:
fixed